blocks_vertical/data.js

```diff
@@ -486,6 +486,40 @@ Blockly.Extensions.registerMixin('contextMenu_getVariableBlock',
   Blockly.Constants.Data.CUSTOM_CONTEXT_MENU_GET_VARIABLE_MIXIN);
 
 /**
+ * Mixin to add a context menu for a data_list block.  It adds one item for
+ * each list defined on the workspace.
+ * @mixin
+ * @augments Blockly.Block
+ * @package
+ * @readonly
+ */
+Blockly.Constants.Data.CUSTOM_CONTEXT_MENU_GET_LIST_MIXIN = {
+  /**
+   * Add context menu option to create getter block for the loop's list.
+   * (customContextMenu support limited to web BlockSvg.)
+   * @param {!Array} options List of menu options to add to.
+   * @this Blockly.Block
+   */
+  customContextMenu: function(options) {
+    if (!this.isCollapsed()) {
+      var listOfLists = this.workspace.getVariablesOfType('list');
```

Comment:
This is the same as the function above except for 'list' vs '' on this line.  It looks like you want a function factory instead of duplication.

See #945 for an example: 
https://github.com/rachel-fenichel/scratch-blocks/blob/f8723d0dccb8a9746f78a503b84407bc56725a71/blocks_vertical/vertical_extensions.js#L152

https://github.com/rachel-fenichel/scratch-blocks/blob/f8723d0dccb8a9746f78a503b84407bc56725a71/blocks_vertical/vertical_extensions.js#L44

https://github.com/rachel-fenichel/scratch-blocks/blob/f8723d0dccb8a9746f78a503b84407bc56725a71/blocks_vertical/vertical_extensions.js#L58